### PR TITLE
BaSP-MR-49: Admins methods

### PR DESCRIPTION
**Pull request by LucianoStradiot**
I work on the put, post and delete methods for admins with his corresponding functions and connections to the database

### CODE REVIEW

File: src/Components/Admins/index.js [hunk 1]
```diff
@@ -1,10 +1,314 @@
...
+        setFormChange({
+          firstName: '',
+          lastName: '',
+          dni: '',
+          phone: '',
+          email: '',
+          city: '',
+          password: ''
+        });
+        alert(createdAdminsData.message);
```
Comment (GaloDurante) [inline src/Components/Admins/index.js:71]: you have to add getAdmins() to refresh the new data in the frontEnd

File: src/Components/Admins/index.js [hunk 2]
```diff
@@ -1,10 +1,314 @@
...
+            firstName: '',
+            lastName: '',
+            dni: '',
+            phone: '',
+            email: '',
+            city: '',
+            password: ''
+          });
+          setIdStatus('');
+          alert(updatedAdmin.message);
```
Comment (GaloDurante) [inline src/Components/Admins/index.js:112]: same here

File: src/Components/Admins/index.js [hunk 3]
```diff
@@ -1,10 +1,314 @@
...
+          city: '',
+          password: ''
+        });
+        alert(createdAdminsData.message);
+      } else {
+        const resp = await createdAdmin.json();
+        alert(resp.message);
+      }
+    } catch (error) {
+      console.error(error);
```
Comment (GaloDurante) [inline src/Components/Admins/index.js]: replace the console.log() with an alert or remove it

File: src/Components/Admins/index.js [hunk 4]
```diff
@@ -1,10 +1,314 @@
...
+          });
+          setIdStatus('');
+          alert(updatedAdmin.message);
+        }
+      } else {
+        const resp = await updatedAdminRes.json();
+        alert(resp.message);
+      }
+    } catch (error) {
+      console.error(error);
```
Comment (GaloDurante) [inline src/Components/Admins/index.js]: same here

File: src/Components/Admins/index.js [hunk 5]
```diff
@@ -1,10 +1,314 @@
...
+  };
+
+  const save = () => {
+    updateAdmins();
+    formInvisible();
+  };
+
+  const create = () => {
+    formVisible();
+    addVisible();
```
Comment (GaloDurante) [inline src/Components/Admins/index.js:143]: set all the values ' ' for the formChange

**Review by Eugeniagassman — APPROVED** [review]

**Review by leimeter-joaquin — APPROVED** [review]

**Comment by LautaroVercellini** [comment]
When editing an admin, if you confirm the update with no changes to the fields, an alert opens to confirm that the admin was successfully updated without changes. However, when you press "Acept," the server fails and the page goes blank.

**Comment by LautaroVercellini** [comment]
Additional, the format of the form does not adhere to the expected navigation. The fields are displayed one below the other instead of side by side. Additionally, if the user tries to navigate through the form by pressing "TAB", when they reach the phone field, the navigation continues to the cancel button.

**Pull request merged**